ppdet/modeling/backbones/efficientnet.py

```diff
@@ -88,37 +96,44 @@ def get_model_params(scale):
         'b5': (1.6, 2.2),
         'b6': (1.8, 2.6),
         'b7': (2.0, 3.1),
+        'l2': (4.3, 5.3),
     }
 
     w, d = params_dict[scale]
 
     global_params = GlobalParams(
+        blocks_args=block_strings,
         batch_norm_momentum=0.99,
         batch_norm_epsilon=1e-3,
+        drop_connect_rate=0 if scale == 'b0' else 0.2,
```

Comment:
would make more sense to split the drop connect part as a separate PR, as it is actually  unused for d0.